apps/mobile/src/navigation/linking/index.js

```diff
@@ -76,6 +76,7 @@ export const routingConfig = {
   '/:context(my)/:myHome(interactions)':                                  `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Stream`,
   '/:context(my)/:myHome(mentions)':                                      `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Stream`,
   '/:context(my)/:myHome(posts)':                                         `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Stream`,
+  '/:context(my)/:myHome(stream)':                                        `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Stream`,
```

Comment:
Good call! As this is a temporary and integral route, and not something we have on our source-routing on Web, please put a comment above it noting that and why it is here. Also was not expecting that you'd put it under the "myHome" named param. Is there a reason we need to do that here? myHome is a bit of a special case with logic attached downstream so I was thinking we wouldn't name it or give it a unique janky temporary name so we don't end-up with myHome logic embedded somewhere that breaks later? Not too worried, just sayin.

Reply:
If anything, I think this is "skating to where the puck is going"; where web will (could) end up anyway. Already reverted the :myHome-ing of this before I saw this comment

Reply:
Well for web though we came to the conclusion, at least for now that this route will be `my/groups/*` which I think is good. So the comment really will help. There are no other routes in the table that are internal or not mapped to Web, and we'd like to keep it that way.

Reply:
Oh sorry, I took your remark to be saying that your work was done here:) I see now you were speaking of the query variables slug thing. Yay. Ok, sail on